sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java

```diff
@@ -101,19 +100,10 @@ private static OkHttpClient buildClient(Builder builder) {
       .supportsTlsExtensions(true)
       .build();
     okHttpClientBuilder.connectionSpecs(asList(tls, ConnectionSpec.CLEARTEXT));
-    okHttpClientBuilder.sslSocketFactory(createSslSocketFactory());
```

Comment:
I don't see how this change relates to the PR and doesn't look like quality flaw to me